### PR TITLE
Added the way of creating non-secure APIs.

### PR DESCRIPTION
**Pull request by Meruja**

### CODE REVIEW

File: en/docs/learn/design-api/create-api/create-a-rest-api.md [hunk 1]
```diff
@@ -124,7 +124,10 @@ Follow the instructions below to create a REST API using the basic flow.
      | `cookie` | Operations can also pass parameters in the Cookie header, as `Cookie: name=value`. Multiple cookie parameters are sent in the same header, separated by a semicolon and space.                                                                                            |
      | `body`| An arbitrary amount of data of any type is sent with a POST message.                                                                                                                                |
 
-4. Optionally view the API definition.
+    !!! note
+            APIs can invoke without using an Access Token. To create non-secure APIs, you have to create an API with the resource having **security: off** .
```
Comment (malinthaprasan) [inline en/docs/learn/design-api/create-api/create-a-rest-api.md:128]: Shall we please explain this properly?

**Review by malinthaprasan — CHANGES_REQUESTED** [review]

File: en/docs/learn/design-api/create-api/create-a-rest-api.md (hunk 1 above)
Comment (tharikaGitHub) [inline en/docs/learn/design-api/create-api/create-a-rest-api.md:128]: @Meruja Let's add a screenshot as well.

Comment (tharikaGitHub) [inline en/docs/learn/design-api/create-api/create-a-rest-api.md:128]: Let's explain what non secure APIs are. And if a user wants to make a particular API resource non secure, let's explain this is the procedure to follow.

**Review by tharikaGitHub — CHANGES_REQUESTED** [review]

**Review by shaniR — APPROVED** [review]

File: en/docs/learn/design-api/create-api/create-a-rest-api.md (hunk 1 above)
Comment (shaniR) [reply en/docs/learn/design-api/create-api/create-a-rest-api.md]: can you please check the modified content and let me know if you think we still need a screenshot?

Comment (shaniR) [reply en/docs/learn/design-api/create-api/create-a-rest-api.md]: @malinthaprasan  can you please check the modified content and let me know if the explanation is sufficient?

**Review by malinthaprasan — APPROVED** [review]

**Review by tharikaGitHub — APPROVED** [review]

**Pull request merged**